migrations/versions/0303_populate_services_org_id.py

```diff
@@ -0,0 +1,59 @@
+"""
+
+Revision ID: 0303_populate_services_org_id
+Revises: 0302_add_org_id_to_services
+Create Date: 2019-08-06 09:43:57.993510
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.sql import text
+
+revision = '0303_populate_services_org_id'
+down_revision = '0302_add_org_id_to_services'
+
+
+def upgrade():
+    conn = op.get_bind()
+    results = conn.execute("select service_id, organisation_id from organisation_to_service")
+    org_to_service = results.fetchall()
+    for x in org_to_service:
+        sql = """
+            UPDATE services
+               SET organisation_id = :organisation_id
+            WHERE id = :service_id
+        """
+        conn.execute(text(sql), service_id=str(x.service_id), organisation_id=str(x.organisation_id))
+        history_sql = """
+            UPDATE services_history
+               SET organisation_id = :organisation_id
+            WHERE id = :service_id
+              AND version = (select max(version) from services_history sh2 where id = services_history.id); 
```

Comment:
what `sh2` means here?

Reply:
It's an alias for the table - so when I say services.history it knows I'm talking about the outside table